src/main/kotlin/app/revanced/patches/youtube/layout/hide/general/patch/HideLayoutComponentsPatch.kt

```diff
@@ -55,7 +55,7 @@ class HideLayoutComponentsPatch : BytecodePatch(
                 "revanced_hide_expandable_chip",
                 StringResource(
                     "revanced_hide_expandable_chip_title",
-                    "Hide the expandable chip under videos"
+                    "Hide expandable chip under videos"
```

Comment:
This is incorrect grammar. Either you include "the" or say "chips". If you do the latter other references such as the key have to be renamed to "chips", including on the integrations side.